include/range/v3/range/access.hpp

```diff
@@ -102,7 +105,7 @@ namespace ranges
 
             template<typename R>
             constexpr auto CPP_fun(operator())(R &&r) (const
-                noexcept(noexcept(impl<R>{}((R &&) r)))
+                noexcept(noexcept(impl_v<R>((R &&) r)))
```

Comment:
This change pessimizes compile time. I'd prefer to see it conditional on `#ifdef RANGES_WORKAROUND_GCC_FOO`, where `FOO` is either a GCC bug number or `UNKNOWN1` (with a corresponding range-v3 issue to minimize and file a bug). `RANGES_WORKAROUND_GCC_FOO` should be defined in `detail/config.hpp` only when `__GNUC__ >=9 && defined(__cpp_concepts)`.